README.rst

```diff
@@ -328,7 +328,7 @@ In the `serving` module you can find an API for creating/starting/stopping/updat
   model.export(script_path, "irisClassifier")
   if serving.exists("irisClassifier"):
       serving.delete("irisClassifier")
-  serving.create_or_update(script_path, "irisClassifier", serving_type="SKLEARN", model_version=1)
+  serving.create_or_update(script_path, "irisClassifier", model_server="FLASK", model_version=1)

```

Comment:
I'm fine with this change, but it also breaks examples in here so we need to fix them https://github.com/logicalclocks/hops-examples/blob/master/notebooks/ml/End_To_End_Pipeline

Reply:
Also did we add an example for kfserving in hops-examples?